scripts/zones/Mog_Garden/TextIDs.lua

```diff
@@ -2,16 +2,16 @@
 
 -- General Texts
 ITEM_CANNOT_BE_OBTAINED  = 6379; -- You cannot obtain the item <item> come back again after sorting your inventory
-ITEM_CANNOT_BE_OBTAINEDX = 6379;
+ITEM_CANNOT_BE_OBTAINEDX = 6378; -- (FIXME: Is this right??) You cannot obtain the <item>.
```

Comment:
Any insight on what this should actually be is appreciated, I just had to take a guess. Even if Mog Garden's not implemented yet, it'd still be good to know exactly what text this is supposed to be.